core/logger/logger.go

```diff
@@ -191,11 +242,16 @@ func (l *Logger) InitServiceLevelLogger(serviceName string, logLevel string) (*L
 	newLogger := *l
 	newLogger.SugaredLogger = zl.Named(serviceName).Sugar().With(l.fields...)
 	newLogger.fields = copyFields(l.fields)
+	if orm := newLogger.Orm; orm != nil {
+		err := orm.SetServiceLogLevel(ctx, serviceName, ll)
+		if err != nil {
+			return nil, err
+		}
+	}
```

Comment:
Embedding this here reduces the API surface and eliminates redundant code, but only one of the two callers of `InitServiceLevelLogger` was doing it explicitly, so this may not be correct. The other caller is new though, so omission may have been an oversight: `core/chains/evm.newChain`.

This is also the only remaining method that mutates the logger, so finding another way around this would be good.

Reply:
Reverted in favor of `GetORM()` method.